docs/developer/kiosk.md

```diff
@@ -0,0 +1,106 @@
+
+## Getting started 
+
+The default Kiosk contains an initial set some of our most popular games from Arcade. This page is located at https://arcade.makecode.com/kiosk.
+
+![Starting screen](/static/developer/kiosk/select-a-game.png)
+
+## Clearing the Kiosk
+
+To have students start with an empty (clean) Kiosk, so that they won't be tempted to just strart by playing random games, you can add the clean flag to the url:
+
+https://arcade.makecode.com/kiosk?clean=1
+
+If there are no games uploaded yet, your kiosk will look like the following:
+
+![No games uploaded yet](/static/developer/kiosk/no-uploaded-games.png)
+
+If you have only uploaded one game, you will see three copies of that game. This gives the illusion that you can move to the next or previous game even though it's the same one.
+
+![Just one uploaded game](/static/developer/kiosk/single-uploaded-game.png)
+
+When you upload more games, you will see them in the list as you scroll through.
+
+![Multiple uploaded games](/static/developer/kiosk/multiple-uploaded-games.png)
+
+## Locking the Kiosk
+
+If you are just looking to put games on display and not to edit the game list already in the kiosk, there is a **lock** flag that can be added to the url. This makes the **Add your game** button disappear. https://arcade.makecode.com/kiosk?lock=1 
+
+![Locked kiosk](/static/developer/kiosk/locked-state.png)
+
+You can put your Kiosk in a locked and clean state by using both flags together like the this:
+
+https://arcade.makecode.com/kiosk?clean=1&lock=1
+
+The order of the flags does not matter.
+
+## Kiosk controls 
+
+The Kiosk is added to improve the "console style" experience and the intention is that users will have a keyboard, connect their computer to a shoebox controller, or build themselves an arcade cabinet. You can do any of those projects with our help here: 
+
+* [Making an arcade cabinet](https://arcade.makecode.com/hardware/raspberry-pi/wooden-cabinet)
+* [Making a shoe box controller](https://arcade.makecode.com/hardware/shoebox-controller)
+
+There's a full list of do it yourself projects at:
+
+* [Do it yourself hardware](diy-hardware)
+
+Because of this, the only way to interact with the kiosk is through keyboard controls. 
+
+![Keyboard controls](/static/developer/kiosk/keyboard-controls.png)
+
+Two additional keyboard controls are:
+
+* **Spacebar**: Select
+* **ESC**: Exit
+
+## Adding a game to the Kiosk 
+
+Use the up arrow or trigger your joystick in the upwards direction to select the **Add your game** button.
+
+![Starting screen](/static/developer/kiosk/select-a-game.png)
+
+Use the space bar or the a button to select the button. Then, scan the QR code on the Kiosk.
+
+![Add your game screen](/static/developer/kiosk/add-your-game.png)
```

Comment:
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/49178322/213318508-73fbc43a-969c-4ce0-ab55-9a4247b17d30.png">

This one isn't _as_ important because everything looks pretty much the same, I just realized I screenshotted with localhost and we should probably have the live url in the screenshot.